### ISSUE TITLE
Export the loss curve during training.

### ISSUE THREAD

**Issue by www11111156** [livekit/livekit-wakeword#85]
Hello, is there a way to export the training loss curve and the validation curve?
